webpack.config.js

```diff
@@ -8,16 +8,16 @@ module.exports = (env) => ({
   devtool: env.dev ? "eval-cheap-module-source-map" : "eval",
   watch: env.dev,
   entry: {
-    admin: path.resolve(__dirname, "src/js/admin/admin.js"),
-    collect: path.resolve(__dirname, "src/js/collect/collect.js"),
-    downloadData: path.resolve(__dirname, "src/js/downloadData.js"),
-    home: path.resolve(__dirname, "src/js/home/home.js"),
-    login: path.resolve(__dirname, "src/js/login.js"),
-    passwordRequest: path.resolve(__dirname, "src/js/passwordRequest.js"),
-    passwordReset: path.resolve(__dirname, "src/js/passwordReset.js"),
-    register: path.resolve(__dirname, "src/js/register.js"),
-    userAccount: path.resolve(__dirname, "src/js/userAccount.js"),
-    verifyUser: path.resolve(__dirname, "src/js/verifyUser.js"),
+    admin: path.resolve(__dirname, "src/js/admin/admin.jsx"),
+    collect: path.resolve(__dirname, "src/js/collect/collect.jsx"),
+    downloadData: path.resolve(__dirname, "src/js/downloadData.jsx"),
+    home: path.resolve(__dirname, "src/js/home/home.jsx"),
+    login: path.resolve(__dirname, "src/js/login.jsx"),
+    passwordRequest: path.resolve(__dirname, "src/js/passwordRequest.jsx"),
+    passwordReset: path.resolve(__dirname, "src/js/passwordReset.jsx"),
+    register: path.resolve(__dirname, "src/js/register.jsx"),
+    userAccount: path.resolve(__dirname, "src/js/userAccount.jsx"),
+    verifyUser: path.resolve(__dirname, "src/js/verifyUser.jsx"),
```

Comment:
Why do we still need this webpack file if we're using vite?

Reply:
As for jsx so that bundler knows about them that these are JSX files that need further processing plus a without having to specify them individually, plus a convention too.